plugins/chat/app/controllers/chat/api/channels_invites_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Chat::Api::ChannelsInvitesController < Chat::ApiController
+  def create
+    with_service(Chat::InviteUsersToChannel) do
+      on_success { render json: success_json }
```

Comment:
Just confirming my understanding of services:

this line is here just to be explicit about the success path, but could be omitted thanks to the default success action?

Reply:
yes indeed